src/mediaelement.js

```diff
@@ -59,7 +59,7 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
     },
 
     isPaused: function () {
-        return this.media.paused;
+        return this.media && this.media.paused;
```

Comment:
@oroshnivskyy I'm not even sure we are taking the right approach hiding all these possible errors in user's application. Why can't the user destroy the instance cleanly in their code? And not call `isPaused` when they know it's destroyed. Or is it called internally?